### PR TITLE
fix a bug when /usr/local/bin is available

### PR DESCRIPTION
**Pull request by MariaSemple**
### Intent

Fix failing pro builds.

### Approach

Sometimes `/usr/local/bin` is available. In that case, we don't set an install dir, and the last line fails because `/sentry-cli` doesn't exist. This attempts to fix that by setting a `SENTRY_PATH` variable which is either `sentry-cli` or `${INSTALL_DIR}/sentry-cli`.

Another way we could fix it is to always install sentry to `${INSTALL_DIR}`.

### Automated Tests

Build system failure.

### QA Notes

Build system failure.

### Documentation

None, build system failure.

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: dependencies/common/install-sentry-cli [hunk 1]
```diff
@@ -30,6 +30,9 @@ fi
 # specify a version
 export SENTRY_CLI_VERSION=2.9.0
 
-curl -sL https://sentry.io/get-cli/ | bash || true
+curl -sL https://sentry.io/get-cli/ | bash || trueSENTRY_PATH="sentry-cli"
```
Comment (skyeturriff) [inline dependencies/common/install-sentry-cli:33]:
```suggestion
curl -sL https://sentry.io/get-cli/ | bash || true
SENTRY_PATH="sentry-cli"
```

**Review by skyeturriff — APPROVED** [review]
I think just one small typo above, otherwise LGTM!

**Review by cm421 — APPROVED** [review]
Once Skye's fix is pulled in LGTM!

**Comment by technocrat** [comment]
In community RStudio, the console disregards the user's $PATH and can find nothing in `/bin` or `/usr/bin`. This causes installs to fail, for lack of `tar` and knitting to fail for lack of `pandoc`, so please be aware in anything having to do with setting a $PATH for purposes of initialization doesn't continue to do this.

To reproduce, use

```
Sys.which("tar")
```

**Comment by MariaSemple** [comment]
Thanks for your comment @technocrat! We are aware of the issue regarding adding `/usr/bin` to the path, but this change is only related to our build system. It doesn't affect the behaviour of the product at all.

**Pull request merged**